libs/cypress-indexeddb/src/lib/open-database.ts

```diff
@@ -1,3 +1,14 @@
+/**
+ * Create a database connection
+ *
+ * @remarks The `createDatabaseConnection` opens a connection with the provided `databaseName` and `versionConfiguredByUser`.
```

Comment:
`databaseName` and `version` (the parameter name is for developers who will touch this code in the future. It signals that this version number was passed down by the user and not incremented by our library